src/reporters/errors/templates.js

```diff
@@ -33,6 +33,12 @@ function getDiffHeader (err) {
     return '';
 }
 
+function getScreenshotStr (err) {
+    return err.screenshot ?
+           dedent`<span data-type="screenshot">Screenshot: </span><span data-type="screenshot-path">${err.screenshot}</span>` :
```

Comment:
No need to dedent here, since it's single line. I'd prefer to wrap both text and path into the `<div>` with attribute `screenshot-info`. And we should use `a` for the path